optional_plugins/varianter_yaml_to_mux/avocado_varianter_yaml_to_mux/mux.py

```diff
@@ -181,9 +181,6 @@ def __iter__(self):
         if self.root is None:
             return
 
-        # TODO: Remove when 36lts is discontinued
-        if not hasattr(self, "variant_ids"):
-            self.variant_ids = self._get_variant_ids()
```

Comment:
The function `_get_variant_ids` is used only once, perhaps it'd make sense to get rid of it completely and embed it into `initialize_mux`, what do you think?

Reply:
I don't have a problem with that.  Adding that change in a new commit.